+1171 91033fc852b8f3e5
+1172 d40e236323f9215b
+1173 a5c17afd34497eea
+1174 148b1f52d933a788
+1175 8856da24fb373cde
+1176 9493c5f6f0f70d1d
+1177 9df37642e8c925b8
+1178 93863bb8005cfb68
+1179 75614d66cbf03cd8
+1180 56e38dd7e00fc876
+1181 3e1e6a823e238518
+1182 63a9ae02dc16045b
+1183 131b5b7deb81c720
+1184 af95e1bba560fc8b
+1185 525ee7ada0f5aa93
+1186 0d35bbbed2852fc5
+1187 a205fe3899fd2378
+1188 60b4d17ccd82cdb9
+1189 ecb28ea647b9c598
+1190 2436f6f03936cf3d
+1191 8b962a73fb3e2fcc
+1192 47a8312c03703d55
+1193 2a1fa988d1b6f8b3
+1194 2423abd7217e114c
+1195 7fc24e80de13efe1
+1196 826767875dc4fe11
+1197 253834e2a6917c6d
+1198 a8d8966320bc567b
+1199 3bf95bfb614fb70e
+1200 ba94a732483a08f1
+1201 6f51ddf2fbfb933a
+1202 f95610f412b76f0b
+1203 6faef3769fe1b096
+1204 b3cebdcf1bc08454
+1205 84bf921c4a48a86d
+1206 5d9fc34da6f4804d
+1207 521f3305d965c096
+1208 60db12727cd82a36
+1209 1821eb887633ce90
+1210 33479e8a89c04d1b
+1211 53b325f0b1126b22
+1212 9d241ecd2afbe84f
+1213 6bb153617dc10071
+1214 69c1e8d050b35b35
+1215 4a923162651b6e4c
+1216 ffb0f004ee563531
+1217 5421c113776313d8
+1218 565ed0f1cfef6d2a
+1219 7a7a76f5138806b8
+1220 69612d54f026ca09
+1221 f3ae668c9e7ebfc2
+1222 82ec3e96e6005ff9
+1223 32d056cc49ce15ad
+1224 ee86b092263adf10
+1225 102e084264f30347
+1226 48cfffadd03190fc
+1227 212eac2872991b33
+1228 d0f8045928a37c63
+1229 57866bec03e9ef56
+1230 e37c124d58cea14a
+1231 18d7d3c3fad4f0b1
+1232 0a63f82849dd5ede
+1233 a5464d21e1cce819
+1234 3da5f70540a467ce
+1235 28e6d01ded164bd9
+1236 c051099da26fab36
+1237 1458015d36f23171
+1238 0ef2b8e68562f984
+1239 42d5023dc719da13
+1240 9db14e2d181e92ef
+1241 a59d0ba2681573c2
+1242 47d86f2033b64cbc
+1243 c175b42104a7b5dc
+1244 58a5c75bb3583d52
+1245 5a296c2f8af62d75
+1246 0f539027ee8d8afe
+1247 6235ade4e033dcc4
+1248 15949d6451557d74
+1249 9aa05c59bd9f4dfc
+1250 e6acb6ccd1ca6f45
+1251 e73e64f615717562
+1252 9ea8d3eb8d1ae4ef
+1253 e0ebcead202ef603
+1254 da6b373ce3386e35
+1255 bbcf02fdb297b341
+1256 09775c89005c097c
+1257 612de941758a2a47
+1258 3dfa7e1a6724485b
+1259 09d1ba0750a9b2dc
+1260 5088e6a6996455ba
+1261 357ca087f7231226
+1262 316c443506e73708
+1263 7d4c15c7af77ba41
+1264 5a8660ca36a99a4a
+1265 aa75ff0958d523ff
+1266 02456d9ac722e8f0
+1267 7e4b160da34f573c
+1268 973319d34b489834
+1269 dda4e8fd9825ad35
+1270 6da1651bee6f64af
+1271 7ace826af5aeccb2
+1272 17f4301c887db6eb
+1273 09e1d4960d332307
+1274 4d4c929c8f6d94e2
+1275 7ce6c739b89ff367
+1276 763d0b02d13d7c30
+1277 c61674b103b97c64
+1278 1856f4d8522d2b75
+1279 2169578210b50880
+1280 438a87238d7cb806
+1281 6dbed1b49ed4538b
+1282 eebe0c607ffbec2b
+1283 371c1bbb10c1852f
+1284 b18390d8e163272b
+1285 541dce24868c2a93
+1286 a88d568023e27bb2
+1287 086cf1fd37decd95
+1288 ac62e281de7d7e84
+1289 109fc81df6dab86c
+1290 2564106bfcdf735b
+1291 8c6530803010cf54
+1292 b85aa2d932a65055
+1293 aab0cbf13c825390
+1294 d64b02bc899572fb
+1295 3d0f278ffb809fde
+1296 ad56693908f9329d
+1297 0778480977141983
+1298 2194fbb3591a2abb
+1299 64758342476a0e22
+1300 5ab7d1ca34a17649
+1301 ee7a4152a77ec300
+1302 22bf64189b236bac
+1303 de19ba2e70b839a1
+1304 4cd26d52ba2f2436
+1305 2974cf33a75e3f12
+1306 d42c76bb99e42040
+1307 2f3e3c2983591c2e
+1308 aa989e1566fb8256
+1309 e6a1554dc42ff866
+1310 37e909c7ae776786
+1311 bfa74e3541a9cb11
+1312 001a038b1e8c5052
+1313 c6941a8fdd032c2d
+1314 e3b683b07b97527f
+1315 3f2729aa21832712
+1316 cc1761999c6fe9af
+1317 3f23484341afa941
+1318 d785c55d02369ee1
+1319 55afa1245af911c1
+1320 e4e92ab4c1e6fa66
+1321 7be3452e50f7211b
+1322 5138fee67ac634d1
+1323 d0a609f07c3a3d13
+1324 02b11eed297a4936
+1325 067217131c7db910
+1326 42a47f12a13b98aa
+1327 0d4788ee35b98f53
+1328 04b4e9e7b39c454a
+1329 29185a2c3c3e974f
+1330 40b487c9eff5dcde
+1331 32f794d2b5eaa5c8
+1332 141d23e23daea039
+1333 42f4fdcf6ad0afdb
+1334 3098a6d5f09f19de
+1335 924e79ed04cba760
+1336 6fa62e2a54f7f5df
+1337 4c65aa32c2c0f169
+1338 0e0a235a79908993
+1339 8e68a310bca80a30
+1340 c855444b1304ab98
+1341 92d4994bee6e42df
+1342 74d92efe7b6ad752
+1343 836ae76c2a894068
+1344 a7a8240f88b20ed9
+1345 6beac1788ab3e31b
+1346 a1792ff5d69a60e2
+1347 6522ef860d8a9b66
+1348 3d20c0f6e69c12a4
+1349 7a49f88a50fa6c07
+1350 78f459c45068fe99
+1351 cb0e41f993291819
+1352 a75fdfb40ae40fe5
+1353 c1ef30e0ab221d04
+1354 3110adcf279e322e
+1355 66c352cc38740e6b
+1356 167453690c724bf5
+1357 cbd56b834df7df0f
+1358 18a133cdc20c1eb2
+1359 ca156d77cf8d7eb9
+1360 5e04bf8cb52c81d3
+1361 3f3e48bddac6aeca
+1362 c221917d2df2b1f7
+1363 42b37ac339c38163
+1364 b22332ffb67e689c
+1365 04fc28032f9f7130
+1366 ae1edc2b0bcebea9
+1367 b816177bc900b374
+1368 110d1939c2f181e6
+1369 d877eb5fa6277baa
+1370 9544565544257b76
+1371 e8de2ab12b43849b
+1372 ea45feaf8c6e922c
+1373 440f7d01f9372d6a
+1374 8a6541001040882e
+1375 1911a99db8794ef5
+1376 2d72b1b600e38ea4
+1377 eec8e3b8f38f57b7
+1378 a6f85718ad3706ed
+1379 b4960b05745e0666
+1380 ded8d3f52a67ac84
+1381 09c2db65c95388bc
+1382 8ed2283dee03edef
+1383 f5eea9b894c403c7
+1384 445961f30d34f2f3
+1385 f5570a792be9a1d0
+1386 a972524a731300d1
+1387 0547a11f56860066
+1388 c8c3d157df4445b9
+1389 a294bfeb59c8f0b7
+1390 85ce5b500abdbb06
+1391 2e71356edbc777b2
+1392 ed7519ac02a23bfc
+1393 e74beee87ab558e2
+1394 cae676f2fd8deb61
+1395 bcf89b4a0e9f20cd
+1396 2a24cfc7e096ef65
+1397 dd34b22d8d0c15cf
+1398 dcf2dd0d3f43a76d
+1399 ba6b5e12c617517e
+1400 db2af180c04cd389
+1401 68a86358ed746ee6
+1402 710ae3f0a0ae7dc8
+1403 7658e287cb3fd130
+1404 5d092b4163c1d0ca
+1405 04fae6cbb1b11900
+1406 ebcae83363fded5a
+1407 845c0fa40b8a0b41
+1408 b669652c39a0b4d7
+1409 6358d987ee35959f
+1410 e68c2643f3f28df2
+1411 b60593e2252e25cf
+1412 1bd619db1a753477
+1413 d59bbaf1f17f620b
+1414 9fe3a36a4617510f
+1415 4689af9a057361ca
+1416 df6e37140367e025
+1417 ba48d349aaea73ca
+1418 f05f64e5b66477b3
+1419 547a1bb78893f735
+1420 2f62ff0ea085ba07
+1421 6f881eb7c8c1382e
+1422 f31e2a1470343741
+1423 f88bdd207a3671c6
+1424 b8d8e1eb6403667f
+1425 e40aa16d6ad88303
+1426 d23b3272d43b2f31
+1427 4b0206a26eed931c
+1428 4b89966313c14ec8
+1429 20eab35aae94b8c8
+1430 cc643ea1be4d2f2a
+1431 2eb92a0d397823b9
+1432 30060b8acb6bf52e
+1433 11f85218b17ab7b5
+1434 afa34f71a19fb047
+1435 f69c63ac042e5d62
+1436 450f3dce5fce9ac7
+1437 7d28bc4391620993
+1438 5a236975c652934e
+1439 f53af7e178eaee07
+1440 e74655c651267d68
+1441 8d36a8eca388c432
+1442 51d5b94061ccbc36
+1443 bc3badd1d7d44b8d
+1444 4e3982943787f119
+1445 10fb2e3c6ff48d09
+1446 6f2f62ff60c01487
+1447 56acf27c73eb4d1e
+1448 e625a021af6a543e
+1449 1e64ee5be4ca4ea2
+1450 429dabd69c9da94c
+1451 ce4343909e923959
+1452 6be9484212ddacdf
+1453 c4eeaeb2b53b748a
+1454 29051f5e7be2a79f
+1455 8e2c5a0ab66acf0a
+1456 7fe9475bd5b8a357
+1457 b5a5206d7f57d567
+1458 60455aa2d5c0d86b
+1459 843ffc10dee4b371
+1460 bbe45c36df0e3700
+1461 6517738dbdade1e5
+1462 f4785c9a983d6f92
+1463 52c7ee661c335e74
+1464 7f7ccca4ff3afaca
+1465 a739f96d9fdbf59f
+1466 4a0e25b71540ee95
+1467 241b18c7ce688a05
+1468 9572c2a9ebef0de2
+1469 ba1d3a69e210a27b
+1470 8b5b9f99e65816b8
+1471 1774562ddb564e77
+1472 280b337929f774ed
+1473 6b3a6b72aeeb4611
+1474 b18c9e94f07db6fe
+1475 25a1e7f2b946fcb3
+1476 e5201bec9f681abf
+1477 e0c167803e4c9ad2
+1478 9c2c5641611d6b7d
+1479 a229cf89429bf029
+1480 1231e92da5d25a1f
+1481 e0b7a4ca0b51842b
+1482 04241873bdee466c
+1483 233a4cbe17526fb2
+1484 42edad385650c35e
+1485 562577dd472b9824
+1486 396bc4464a4294ff
+1487 e1e08123986443dd
+1488 4eb29de73788a6a3
+1489 815fd0913bbf7e83
+1490 b902c7b6958edee4
+1491 00e6e6eb4b2702b3
+1492 4351fa13b4bc18bf
+1493 eaa376ac6db54927
+1494 ef22d2bc55052c5c
+1495 12df0a6eb286bebf
+1496 e215c2b5cba78571
+1497 ffe448bd3985d6cf
+1498 1f870eee3dab820d
+1499 7b2fca4ce54ff130
+1500 305d647eb66bb346
diff --git a/goldens/qos_contention.log.bin b/goldens/qos_contention.log.bin
index 481d047..166d691 100644
Binary files a/goldens/qos_contention.log.bin and b/goldens/qos_contention.log.bin differ
diff --git a/goldens/qos_contention.t1 b/goldens/qos_contention.t1
index 9c49255..62b7566 100644
--- a/goldens/qos_contention.t1
+++ b/goldens/qos_contention.t1
@@ -3,245 +3,245 @@ t1 1
 demo qos_contention
 seed 4242
 logic_hz 20
-catalog_hash 1106c1cc8652a91f
+catalog_hash a7b85c806df0397c
 ticks 240
-1 35880e54baa13bbd
-2 7d3c210b9cf6080e
-3 f27e502c29d8d1e1
-4 49c9346b7073f43e
-5 159b87c8e7325a9b
-6 976318f7536c4198
-7 6e5ccfd40cd350dd
-8 975c9f2f65e384fa
-9 829e06e11e8c40c7
-10 5a9af91fda3e6027
-11 a834a37a449e25db
-12 4544c7404f93d99f
-13 a42a222127ef4a8c
-14 a626d733bab8aba3
-15 7c6be9972a384fd7
-16 bbf862f6c76008ac
-17 ec0d20e2e5e6065a
-18 fd987728c0607751
-19 a2471043aa8162d0
-20 feb0fa582c0f196b
-21 57abef46303fb1cc
-22 d547c39c51b5d5dc
-23 47d44cb1aad7ce05
-24 03700d4eb5377578
-25 37d7d50a3d8bdc56
-26 7faffe0add999bef
-27 0d91ea13f271fcbe
-28 469b5c2ff60b0488
-29 66aa7edc994ee831
-30 e138a6d43420f63e
-31 103e3edf4922cb67
-32 999eaa2a66f0c96e
-33 a37a4d36623446ed
-34 0dbe6edfd3a4d07d
-35 35ddc847cfc9e12b
-36 e082767758cee0aa
-37 d59de9d2cea63af3
-38 d13e537f6bb77814
-39 36222070e9097fc3
-40 211d1aeeed63ae51
-41 18a78ee0a4a4bc06
-42 70a81565f46436db
-43 1c8a5d40ca7c8040
-44 0a7d16573bd674d4
-45 5cdcdabe4de17dbb
-46 28df05270a6d1937
-47 550c680a5c7fe6fb
-48 43d79373d0958ada
-49 fd6952895cb5ea00
-50 aac89e54bcf64876
-51 d61d774e9c09a641
-52 4ca16829f239f3db
-53 a5a1660b39dffad5
-54 190a5d618393c2ea
-55 784e457574fc28e6
-56 51d96a57e7f694ff
-57 9cc30a5f1be662a2
-58 b49a7b568df6e49a
-59 9647e227f72302f4
-60 4ddde2b122c7e27b
-61 360018cde83966fb
-62 c038719c87fb2c4f
-63 b51675db1f23e966
-64 9d2c4ad3f3b91978
-65 41330738636e82ec
-66 b01a5c437a8a7fb5
-67 34840d0ccd7bd000
-68 62874a49dd3d36f5
-69 edd7ec3818db045c
-70 ad9028873e436958
-71 da6f118467ef1de2
-72 4b055d3e9779a889
-73 81e1f50ee98ef056
-74 2e22d4ec8ca8ecf8
-75 87234c470beb4c23
-76 3e5ed01b49f4a69d
-77 2d2ecd0a41f052bf
-78 5dd0a80bc0dfa0e0
-79 a2393e42c02884e2
-80 757149ff613c8342
-81 20f74c144a1dc27d
-82 c304a48b554ab735
-83 ab37dea29a03e055
-84 5209748e11e29608
-85 9e95c96e0113bdb7
-86 d529799bd5990792
-87 3c4f5cc10bd039fd
-88 9f75c70d209b0af3
-89 ffae333677f7937d
-90 3c303c89f9dbc62e
-91 f5d8ee5ba9d1ba4c
-92 683f27eea7acebf2
-93 752cc3be72247199
-94 4bd29b71e5015165
-95 d0b1e614f95295a1
-96 cd5f2cadebc12fbc
-97 020ea2e675c5343d
-98 f85aae1c709e93d4
-99 d91880eef2672abb
-100 0d129ba86b04e205
-101 d139e684450d2c3b
-102 ac3bcb10038f53e4
-103 6ed3839816b8edf6
-104 813ea8274b763c62
-105 ceacf7aaa8cc51e9
-106 c826f15f331aa459
-107 dfc6ba120c539c7d
-108 fe308bcfe830bcb0
-109 eb3fa5785a8dfe1b
-110 bcdf858dfe0239f9
-111 07c35540653b1104
-112 f065eb8f47368812
-113 5eea950d3ecc31a2
-114 a2da607b3a1a7f0f
-115 6b165a78fb50c7f0
-116 f30f51ad3ef92cfd
-117 a3c344e870e744ec
-118 3441c1d1ab86db00
-119 5a7be48f0d3c84f2
-120 6acc6f8a7c43e179
-121 1224a87a8b43e291
-122 a01dabdcaba8e001
-123 2b8903c157175ee8
-124 839dc3cc45ff8e5e
-125 74c09fe26d9cf072
-126 e06550614dd0674b
-127 406a2231c60e657a
-128 24e9df06f3b8ca73
-129 9f63c987d5c49276
-130 877a128aa252ba2e
-131 2c23e602752c4198
-132 2ac0fd2927b61dd7
-133 06fdccf22cc07ec7
-134 4f9d830f7e3d2f3d
-135 e884b0f9fcc43acc
-136 8937d0ab90e11a8a
-137 ebcffd8ce051e6a1
-138 f1f753f7f396f2f2
-139 3d7ea235a0ac8f73
-140 98817d4f1e497f81
-141 3f14a09709347fed
-142 d95036962b070c01
-143 18aef9bd52c58f87
-144 e7f36d209901895e
-145 586733b928d684c9
-146 a3d306ef812026e7
-147 b98df768f858b8fb
-148 abb9db3be915b4e1
-149 b974257540a33eec
-150 7a954caf27c97bf9
-151 79215dd70b2bd691
-152 38d2bf0eb0ae6f14
-153 f5436f839c91216e
-154 1903c73a92e8087a
-155 afc48b6e6c6c7cd7
-156 098dbbe6166ef426
-157 a38939fcfe8561eb
-158 0a17adb421e7c015
-159 6f72296c2c0c82e5
-160 083a60446966cba7
-161 147d6fa6b060f015
-162 ac36f31172db8a2a
-163 9ff7a22beb1c1854
-164 731002766f039f48
-165 3916e690677b0c7e
-166 a740db40eb4c365e
-167 99c4e5269b3e92bc
-168 578c2a5113c50abf
-169 fdc6eafa6a8797be
-170 78f8f7597d47d3bf
-171 3d488651a829d1af
-172 33b96605425d3225
-173 b42316bbea5b32ec
-174 f9e679b4480702f5
-175 38a7389fc3645f59
-176 2b9f8106b13cc23c
-177 c57b7f6afa07c00e
-178 0ae6b20c5979423e
-179 19ba18f199be044c
-180 2349d4baa7513fff
-181 be73489a646b5c83
-182 9daebbd79c3baee3
-183 f92d9deaa7f586cb
-184 222bf9b0c374c1c9
-185 c45f3bfa390893af
-186 af5d67263391ba1c
-187 1e55418962c6c160
-188 78b504e2a5521ed9
-189 ef5c628b778f891d
-190 05ea3628e94adaf1
-191 7dd6a250e92f07c5
-192 5accb87ad80bf714
-193 022aa9c6edff0a81
-194 9ced9900873ef957
-195 9f08cd830a6c8437
-196 f1d2d68a3cc2227d
-197 8325e834a9b10873
-198 200af8cebc209b6c
-199 fc8158816d54f6fa
-200 225fc4e072503797
-201 c795d905c88d5acb
-202 71539f544a9ea9a3
-203 1764f99090d530af
-204 94e937d3b00b9506
-205 2180537abe47c847
-206 73bf8fb91398acf6
-207 f9c361947f66b7bc
-208 e2f90732e129e8e6
-209 2e7767b29b135a82
-210 9928bd08f93810e3
-211 cb146e73b5641401
-212 ff012d66c0626a0b
-213 f5301ca63781d3c3
-214 a3f24c84fd39ffb3
-215 130919061562f393
-216 4490f7f7b7e483ea
-217 d17d2cafd0d26d43
-218 6695a873f6cf97e6
-219 6042b5a07c1f7404
-220 ced20ce1afe2febe
-221 9f30b69ab7d89afa
-222 43d9e8170d1b14c7
-223 e246553bfaa44e13
-224 795f29b0915c0e43
-225 e3fe57e3c74bbd73
-226 c7682fc0acf503d7
-227 2b1ebcd7f8033c57
-228 1232e2c9dfeda06e
-229 89669ff40ab56982
-230 95fb590311522347
-231 c626286c5ab5465b
-232 c21d9c44a09c0549
-233 4da671034690f983
-234 9767a9014509501c
-235 471642c36d3c25bd
-236 84b131caf7a96307
-237 778d0bf34e4f7383
-238 ad8fc60c82ea1f9b
-239 fb9d5002790a08a7
-240 46006c76c7c337a8
+1 e755104fc8bb8c78
+2 6c5238c43eca210f
+3 1933ee0c6a0b8584
+4 507888df45cfe2ff
+5 f57bb06f8dd0792a
+6 b5f1d43995684665
+7 efd7c12e30742490
+8 f2c241ae0620e97b
+9 7384c9a4c4138236
+10 d86fca44feac7712
+11 4948493fd9f97642
+12 8da5763178c88776
+13 4f0461938bde5d71
+14 c720790ca9920a38
+15 37c2502dcd955ac8
+16 518f900a252964d7
+17 f6fd3cdcb6740e5d
+18 9b21c00bf3d601ce
+19 2874952228521d0d
+20 9f36f3f827d2a716
+21 3065d3ada10b1ef9
+22 2f5fdba1089a6b29
+23 6455afa2a6b1718e
+24 c696bbd3cbb9d451
+25 77f8f05ada4f45a1
+26 2cb2556625f185c8
+27 37956558a75365e7
+28 bb8f49f47b136f35
+29 4d35940befe423ae
+30 501b1f44344bfc2f
+31 fdace5b6e0180bbe
+32 f971b6c975ba14f1
+33 f693225239374d8c
+34 2fa2c8227b3d051c
+35 b7b7cbe3ffd4c020
+36 2864e744d20b19c7
+37 e54a497a2f740364
+38 56db227ac4b2eb6f
+39 883cd4c43f7037ce
+40 876f3e59af5b6a98
+41 cc23c3b3c1b4235d
+42 b53af40788cefbce
+43 eb332d108679cfb7
+44 e9e82b2831a4925b
+45 e58a186a57c5bc26
+46 51d703baa1bc0842
+47 19abd88d7c30572c
+48 ed64e847a58f9cd3
+49 07c0bbf90e85a953
+50 f65cceee6ec2caed
+51 24836c654c350f70
+52 7e4f306d829faf8e
+53 c930fe70b5072dc2
+54 3be3f6a6b3d56783
+55 81a5d19c3e1aba79
+56 b01bdd9cf7b38adc
+57 ad4d8b3bb1424753
+58 0a49ef9bc79512c3
+59 e1b566dc248b00df
+60 f25ac71fa4b54c26
+61 cf5d657ba0f2c31c
+62 bd97b801f1d045f4
+63 62cb88df92c23f1b
+64 0f20ca57901c6311
+65 d1495c60ee6a519b
+66 0d8c39b345d9e610
+67 82f51ef52d00e1f7
+68 102f60a704b65b7e
+69 8b08fca728a07241
+70 7778bf1a7c25caf5
+71 e5288d35f3634191
+72 33ae8c3a7eecb530
+73 3c77abbba8509ad9
+74 030a70a9189f9ce7
+75 af9267bf2ec6eb1a
+76 f6cd72e24cebb550
+77 cc23d8c534214fbc
+78 5db42022dcd00f89
+79 c5fcf7aac5ebae1d
+80 dba92e18f3bf39fd
+81 4f7e75712c906f3c
+82 106a2e284cd5f34c
+83 acb68997c9d62cba
+84 009f6836010c025d
+85 8e40dd105740d9a8
+86 742081da896b3355
+87 72b8d3e7b7c3ad40
+88 9f7c247ed571c3f2
+89 3adcf1ab4db5642e
+90 0916a0e813739393
+91 af289391887fbfd3
+92 dcb405637067ecf1
+93 aebb44fd934b4b1c
+94 b1960004078a9780
+95 8eb3b0cd97a2d91a
+96 3098f214b8ab8a0d
+97 a71e2dbf9c85c4ba
+98 ec8a78c18d1c4b23
+99 688099ec07bbfc72
+100 65edc639cbda1980
+101 d08c9fafe706a658
+102 8d894aa2342020cd
+103 e031a5a88603aef9
+104 5ee07ffdd33c3ed5
+105 a3358615137907e0
+106 29cdd19f82055308
+107 c78164cd3a83483a
+108 8464a1f4132b22cd
+109 104d24a99acc0a94
+110 63e25736aa8283ae
+111 63ecf887d8f18231
+112 934c3ff37b406a33
+113 d94ad6709d532079
+114 48d39df95b88f012
+115 d0b75fb7020c2f87
+116 6cfdc3326a250c06
+117 7f12a27b078dfbf1
+118 b08fb37849fb412d
+119 b15cd74507e166b1
+120 56d3491578fcdb00
+121 80103bd08f060a1e
+122 eb49b383a587416a
+123 44b53c3ebee30df1
+124 385381fb5c3de7f3
+125 dd7a8200cac9b1f5
+126 2bf75f72f6ca27a2
+127 327f2a366ecfd00d
+128 739c246b243b6eb0
+129 0dab8328ff53b377
+130 d3c9b37316abaa47
+131 4fcca002c8358073
+132 bc78bcd03d834b72
+133 e7f3bd95a08c88e8
+134 43cace965bb7c18a
+135 f6fb908f8b498201
+136 e106ddcf6d0b9a1b
+137 591e45bf7592db1a
+138 06b3e241319ea73f
+139 65e12132a7cd3e6c
+140 60b8bd56b9459322
+141 d626fb46a0fbb0d8
+142 0085331b325caaf4
+143 bd49871cdf792400
+144 7bcf00965e48ccff
+145 9d393e4ed705266e
+146 c495d4c686b417a0
+147 dd159a72f4caa9ea
+148 5caae15dc294d784
+149 2c60f99e11f02f67
+150 dace8f991d172c88
+151 327cc63e326daad6
+152 e0a57061397b8e97
+153 b3eae1d66fce231f
+154 3ed94913be16f94b
+155 7e7960c8d2198874
+156 14286506b53f0efb
+157 7f4b0d439af22074
+158 46518960e5eeed62
+159 76857827a33760e0
+160 94dca785fa62e9fe
+161 5f2a10ae9dc71d26
+162 2ef32c05de48f187
+163 32d0e119bf8690b3
+164 9c617487b8c87e27
+165 97a1b642ea7207b3
+166 306db34d15644db3
+167 ca2b00a81a4a5ae3
+168 138a8969dc248646
+169 ca668380ae66eaf1
+170 ddbbbf0be32ed820
+171 fc5e2622ce408626
+172 a9e36d1f5c971eb8
+173 f6e9c22c5a8f1eef
+174 8cf425f0e33daca4
+175 1c0dbbb4a3f3711e
+176 6c3f62677db6f927
+177 a10145a29544b0df
+178 90492dc539f550a7
+179 3948fe33c166594f
+180 8908f33b4984fc4a
+181 8f6d78e93e0a50dc
+182 1a93306ad082c508
+183 92b089ccc0009cbe
+184 23d953954aabda18
+185 ef2c617c97156550
+186 58bc587820116641
+187 9b79696c8dd6579f
+188 dad8489c00ee4b22
+189 8c1226f75b497cc8
+190 021c5f67d5463484
+191 11478cf37532a076
+192 20030bc2fd63b15d
+193 ac436312dbd942d6
+194 4286d3cdd723b640
+195 6e2eb21bada7488e
+196 14228fcbd2a59748
+197 c9c3f488b136ad10
+198 6eecc3c68f169405
+199 6d932f1c13b0206d
+200 dc6dc7d8de34c914
+201 767c55059a6c93d2
+202 64db2cac8ed09202
+203 af05a8d8705d51b0
+204 c1b07ad0385581b7
+205 a677333bc11b07a4
+206 2ca2228f76da676d
+207 bb1bd4479102c8bd
+208 9c609a178be2101b
+209 c325b1e02e77276d
+210 ca2f9ffc4c3c00e2
+211 d69a0f6ee597b44e
+212 8240e16c83ce23f0
+213 a985cdda4f0ce2d2
+214 e58e650943c75bd2
+215 cfb06e1963800198
+216 724900d1f46530b7
+217 de748e0d1db3367c
+218 ecee779791bf3e69
+219 8698b5ca7fdbfa69
+220 aa759d66d6dcab27
+221 92c8b19dee97bd39
+222 7bd56108d67f7b42
+223 4f8358b19aef8a94
+224 fb2c56d7ba9d40fc
+225 54be78d27923bcf6
+226 b527fd9329d47dba
+227 5c4ef5d7721ffa50
+228 8c4acf554cf5c4f7
+229 5bef8e315c8b43bd
+230 d273d465b87a9ee8
+231 70333fae5dd2904a
+232 d52cc09dcdfb7cfc
+233 6e80161188ad51b8
+234 382272dd197e5abd
+235 5e1b16d6bf02b92a
+236 e160c39e1a418724
+237 3535c2cac6b21eea
+238 446ee2d93774e402
+239 fad06399a0d7e7bc
+240 29d1112851993fb5
diff --git a/goldens/qos_emphasis.log.bin b/goldens/qos_emphasis.log.bin
index ac1049a..c32e4d6 100644
Binary files a/goldens/qos_emphasis.log.bin and b/goldens/qos_emphasis.log.bin differ
diff --git a/goldens/qos_emphasis.t1 b/goldens/qos_emphasis.t1
index 2dcec6c..b007545 100644
--- a/goldens/qos_emphasis.t1
+++ b/goldens/qos_emphasis.t1
@@ -3,305 +3,305 @@ t1 1
 demo qos_emphasis
 seed 3003
 logic_hz 20
-catalog_hash 1106c1cc8652a91f
+catalog_hash a7b85c806df0397c
 ticks 300
-1 f0f557190e21e587
-2 7e33fcbff1fc77d8
-3 82e093343c33b098
-4 0abeb8168231d994
-5 aeecb4f7a3219893
-6 3d5623b920bf60d5
-7 f2f7af5bbd007ddf
-8 4310b29fbe23fef0
-9 0a81bd0146238507
-10 20bb1546a5965a45
-11 f0ad03e7bf8e09b1
-12 1f09a289f504a934
-13 66ddf6705d9caa4c
-14 27dcd33c98626484
-15 1524186cab65ac5c
-16 ed619540f86be051
-17 f2d4364d1713f25a
-18 57552135041618c6
-19 727ce61850817ea5
-20 bb2fa913ab914d36
-21 cb24142e273b1d04
-22 cd4d634e6d64b193
-23 f5955b60a37f243a
-24 eea104efc5b36373
-25 41bf7deb4ca2424d
-26 6eefbefda2731335
-27 7a9475d52020619d
-28 4726ffefdbad0fe9
-29 e192e7eb9895132b
-30 7e4def72e4229baf
-31 99d861053bd7c8dd
-32 da29754e17f8dc5e
-33 e53ac9cd456ba796
-34 49a6a6ff910dddca
-35 6a3cfe4b1a723091
-36 a59a7f26b84b5cf7
-37 ffc931ba9d2bd186
-38 5e7aab4f4b14a82c
-39 52648e522b3028f0
-40 35338a8388b73ed9
-41 e2317a16b311646f
-42 379d298b7a1f339e
-43 b511b203afc9701c
-44 2ce45b857e73b2d4
-45 b2a8ac131813c883
-46 04ed71fa3fe260a7
-47 0ac8ba59239c09f1
-48 18f86d268ad03a0a
-49 0a4708ca3840f95a
-50 a81af1944d9f67d2
-51 a7406db8614e1c7e
-52 b95496e4156daa33
-53 78bf0d92ff59f4eb
-54 a6f43817ad1d059d
-55 344e17d26addb5b5
-56 818c18febc32a327
-57 cf7b5b678eca050a
-58 4986c247ac477728
-59 faa63a3c1423500e
-60 1bd8d7381c703d15
-61 5993c6b6ccc16b9e
-62 e81f42cc5d63acc4
-63 d95024da6f8fe4bc
-64 3ecb2950b293ba8b
-65 9618ec0d976ee9a4
-66 a18d343f7dce6c73
-67 8c6e936f908f4b2e
-68 e13b8eef687a3837
-69 fda8550fee3ee564
-70 92b287c423fd800d
-71 cd2b5d0d4e1d9948
-72 734ee4e3199f1934
-73 192190a523318f98
-74 9d1474f194fb63eb
-75 bdc07e02d615feca
-76 d98b743809779774
-77 869ee5cba11d04ee
-78 7c28536deb1ce1eb
-79 9e53d8ed25814779
-80 33a95d5530ba1b5c
-81 94b4f8f3a127476d
-82 c5719fc2bde1689b
-83 2655fbcff59e0dd0
-84 091a9ddaa0b5d874
-85 eb9c95e339319f37
-86 91671cf21bd66883
-87 0fe0368b19b9ce3d
-88 c7bfe13397e74c63
-89 00d37e4c3e6b6e7d
-90 f737fa3b95dbd9fa
-91 058d45c1f4eca2e4
-92 6a47641511d64679
-93 2277137ca06569bd
-94 70efed6870c236dc
-95 5282678a12772098
-96 6af11cdbd31d4e26
-97 cdc78b4beda449fe
-98 a0229c2a46fadd48
-99 1c74b3fe1e20dd7e
-100 192a1fe4116e5a3c
-101 f711ee88aeaacb24
-102 dcfa15d84355bd1e
-103 921efba5fd3e950a
-104 287692d9b703289e
-105 275ca4223183ddaa
-106 77c2e6d1776e80d4
-107 aa804317c02b9d1f
-108 3da762ff98dc2d40
-109 26cf37cf0d5ec06d
-110 b8db23d6697fe61c
-111 2d2a7bc567650371
-112 4a3a72d3d5988929
-113 0a379670702f1197
-114 45fb8630ef0e1f36
-115 abbba2298b342015
-116 b5bb92fbf8ada1b1
-117 4f5326f9e667a9ee
-118 67aa2899eeb41fb4
-119 2d8915731a784a1e
-120 9a64a62368b0aec7
-121 058dc6b8d486dc76
-122 747e307c2f61cf2e
-123 e407dd97ac4ae12f
-124 56fb21df16558964
-125 22e1bafe97a10a77
-126 8c399e66cb42b1f3
-127 051451e5136aba7e
-128 896989fac13bcfb1
-129 836f3913fdc9b5df
-130 59a4f2636192942e
-131 a66d81835f95d01b
-132 4e3ba38037960bd2
-133 f4cc023302184327
-134 c642c9ca3b69ad4e
-135 59aacef7a79e3336
-136 99b152b002b1fe36
-137 70f0b58df4f4b723
-138 df19567e9ac74cfb
-139 0b4aa3d627736b10
-140 f0a2caf4ceee16c3
-141 e90bcbc3bb2f283f
-142 7808fec76df7eb0d
-143 2c6cb9b91c371216
-144 8c0435a0f026a971
-145 49c5ebc69633bbe0
-146 f254b85a8be6ec06
-147 e37e15ce45948cde
-148 e813a1a979f2c6d8
-149 8c0f438ac909088f
-150 97e103a54603fd59
-151 ad07216ff4a35e09
-152 f5614a15303d5923
-153 b441cad782c46bfd
-154 84b117299a04b241
-155 07d5cd465993467e
-156 708f3b0c02ad2d23
-157 afb1897b66693da3
-158 0a43a40b654e93a4
-159 2a1f7c1aa6074e9d
-160 bcd3948b617cf403
-161 d98dfeb00c5c5263
-162 2c6bac158230d98e
-163 786eb94e1b238829
-164 461825c2cea17411
-165 d0343299d7b0ecd1
-166 9d0f8b35bf62952c
-167 7daaff3cc15370e2
-168 1fc25cccc2f54895
-169 e640435721f48585
-170 ffa19f98a50be3f7
-171 b0f07fea57878e49
-172 ebe612813071617c
-173 18a8d968adc08bc3
-174 1421906aacc76ef5
-175 63f93d85178d8c06
-176 b8410bcd712d0469
-177 ad66d07894f91e90
-178 50cbd301b9d59864
-179 da2bcd257540809a
-180 39678e915d50c5bc
-181 00774c4950a19eda
-182 2b45d3bca2de67a1
-183 0e00fdff194e71d2
-184 afb407be46e5997b
-185 420bcc1e5811375e
-186 1d92506f1b3829ef
-187 1c448f1dd5cba60c
-188 cc91a717ca896ad8
-189 2622c3f597bc060d
-190 a4f24d54c4c238a7
-191 8a7e5e8f00aea36f
-192 d60a0182ffb1d55b
-193 5686b1d34000c981
-194 1ccbea8bb942b5c1
-195 6f14004d4dcb98fe
-196 d77694e612f3b3dc
-197 b1f38acd3c1b24b0
-198 293cceb7ff4bb41b
-199 78a8c7d8e9b890f4
-200 0a3d61e410c6029a
-201 21fd0aa388f95b83
-202 2099f3fc42e11914
-203 aa22fcc99631e23c
-204 df00db7e472edefe
-205 d27d20adb5655a92
-206 9db3fc76f5f50e61
-207 8fbbf53645dc2c73
-208 40c8856d3f07ee71
-209 9e82b9c884b2763e
-210 20f3495238259167
-211 64956e5ffaf1a9b5
-212 a3021aea41afa13e
-213 81ee66109b91a7c4
-214 003a9690adb92481
-215 a7895877300874eb
-216 e5f4b54c757be16c
-217 3af8d6b6a4bfaec8
-218 c720f726a9987583
-219 94a2a0b501cfd8bb
-220 d82451b986588b1e
-221 0724003952792132
-222 9f0e87bdf0276966
-223 1f898789e835cd74
-224 94ee4623d4197e43
-225 3bfcf8403a3ea656
-226 ed8af52f875725c0
-227 7e84ce2279d4a726
-228 39a25d69caa43d54
-229 cb5b4e6cf8d58e83
-230 c1acd777eccb323b
-231 cd56cf718fa81f7a
-232 b5488ee007886042
-233 5a3940510a78ac2d
-234 a1d1c6046f7eef98
-235 c4fe1fb333f61b29
-236 f3f98f33b70978c7
-237 1e90153ea52d930d
-238 b0a96a9482af84c6
-239 ac3c2e2e17c79f29
-240 874d3239814d1966
-241 c7a8455129d252d7
-242 97cbd4191f5447d7
-243 410e49a793192c9c
-244 06ea863bb2268f18
-245 073e3e25d83a295b
-246 027c347b68606054
-247 07602ea6db1a6d54
-248 6a835eafe460c3c8
-249 715b927e584939a2
-250 c6169cf4b15d8c3d
-251 f38e68d20d1ca95d
-252 f728edac09bd14e1
-253 65bfacb52ad68bf3
-254 9b745c98079a97c4
-255 ef5019cf169919d0
-256 19d2838e6c1ab8c8
-257 674f3adef504ba1a
-258 b8a25a3bd8e395ee
-259 62776d1eaf184832
-260 a717c5105d2e2ce9
-261 ea2330c6751f869b
-262 ce89a9c6de033b0f
-263 d2c51bb134765518
-264 5c87f9f63831b651
-265 921523035b9487d3
-266 110dcbfc9b6cd734
-267 86185cdaa166cc0e
-268 92ec630da1c61395
-269 03eea5cfe7e6c71a
-270 376f3dd6e25da704
-271 12c1f39fdcd8aa1c
-272 95885d143c8879ae
-273 8130f5325c4053e4
-274 6e503be42cede9ff
-275 a49e440e2f766528
-276 e1790d5fcb82c0cf
-277 4619904be3dec8c6
-278 58578ce78e864b39
-279 3e63e4f8d0f726c8
-280 717efeacadb5470e
-281 60b766ee791cad6e
-282 22f6484f5e98da73
-283 c33ee97287dfd9d7
-284 c1c8fe910f3881f6
-285 a0df3d4ac16734a0
-286 cd4e6446e11ccc85
-287 a0fd22afad83812e
-288 3fa4d5e6119af40d
-289 c9a38c4334bb8fab
-290 1a2683a2c8b888f6
-291 ea730bf7265e5905
-292 c5453648e6da35a1
-293 b5cf970b87d283fc
-294 78134319eb9a25ad
-295 7b818788487d736d
-296 2590e5dc4a5bdd59
-297 ec5ed521c77760de
-298 58c3168401fc3142
-299 bc19c9e849cbe95a
-300 994f029d29260aa9
+1 d00c066d97c5013a
+2 877a34cff5f03011
+3 2b383c9ae91c73ed
+4 b94b7f89514a2a87
+5 338c3d66e09bbbf4
+6 bd69dd10e8132a18
+7 04328acede1f7c26
+8 2edeb5313e962739
+9 844aa379a9d2f602
+10 2e711ebd911585d8
+11 87aeaa1629903f96
+12 aaa1207a16fb4b71
+13 368813b8eb350171
+14 335159f4c8ed3abd
+15 909b3f80587c1455
+16 c51a7642798f3fbc
+17 872d0d4bb4a25755
+18 a88a712e36a2a309
+19 f63baa428cc85876
+20 275d09251a0f6b7b
+21 13b12ae2d90230a1
+22 a2c68b9fddca12c4
+23 856f8b543cbba55d
+24 cb90393292d46ba6
+25 cfeb38656fa2044a
+26 cea9a183aff0fc58
+27 c113b08b69a30220
+28 7a3f750f0904ff76
+29 d7d52610769471f4
+30 e58fe15da5939426
+31 a10cf39dc3974f1e
+32 cbfc689cca4dfa15
+33 e8a900c9bdc1cea9
+34 08edd49b6aa54a81
+35 9ffed1ba37ca150c
+36 5b69ff37d6976f06
+37 36c1b1cb9949839b
+38 6340587788fbcf6b
+39 4b1a14d671be6ba5
+40 ac33912b3713e6e8
+41 aae4507384003e20
+42 bca3d135ffe43769
+43 d2c2648f07d1d2d5
+44 9abeff2928f07333
+45 015b0ac6d2184a90
+46 4f623e11bae020de
+47 199e40e38d9cf1a0
+48 a0cf2a5f17367fb1
+49 617441693d54ee1d
+50 3fe339cf139bb24f
+51 057cbfc0087066d9
+52 ed6c95b34b615060
+53 aca377b0c1237fe6
+54 479f4e249763a816
+55 c771eedd2099e86a
+56 00efc62161e53294
+57 53d1d4c2076ba3fb
+58 6af51dce88d5fc69
+59 7211d06e34aee15b
+60 27c50bc75a4fff0e
+61 f273825a4b0f7d37
+62 e9ba05bdd62dede3
+63 56f853ef6de0c831
+64 3b8bfd434bd72888
+65 81fc7973cb3a739d
+66 ce20ac2fdf46efc6
+67 213ff13a2a276e21
+68 eb7546fa652a84b4
+69 b0c6c42aff56bae9
+70 39a3b4d3c8f85c80
+71 0df87b8380e23885
+72 27273f78a2a42d9d
+73 951edfcfc7f96439
+74 03c4f4cd338c03a0
+75 b968a2a499008ae9
+76 6bcdd865678bc4a7
+77 8c8c7800e1c796e9
+78 d715472c0a31375a
+79 996873985e798a36
+80 3ea4dd07ed42e99b
+81 a11e8343867222ba
+82 8c706535cad7261a
+83 4f35abac9e47fec9
+84 b2e637bb8fdfdaa9
+85 42f8072a84294cbe
+86 7d0a5e23111d92d0
+87 b1d9f87d4d55bfec
+88 fc29788d68291212
+89 ee7e1ed7b997c2f0
+90 73c3d97d701c594b
+91 9d9170ef93c72d1b
+92 08caf5de71e70ac4
+93 5db1bf7b01fac52c
+94 24bca5d83c89596b
+95 6fb0a679e9b12757
+96 a5d055986585a0f3
+97 00bd8f02cc36f1bd
+98 d91678d99e6efe01
+99 3feea16a100055fb
+100 ae1b5dc3c8c34eaf
+101 7b85e1f959f88897
+102 9c464fc5e0e061ef
+103 3eb2aaedc1d97501
+104 18f0f90b29d1393b
+105 fba035fdc164c72b
+106 0230a6443e6fce01
+107 14d5ab9b4e5e06fe
+108 7ee3078e674d0d5d
+109 34192e9150eaee1e
+110 08e785b08d856893
+111 61c5084c6e58675a
+112 db4072c78ec76b48
+113 11131eec60f33992
+114 ce59f661a6cb1d85
+115 0c1f0e950ae2179e
+116 9f2502b806afb088
+117 0d6b24e10f2ec597
+118 cd71b5fae253f7ff
+119 5298e60bcfa86547
+120 8f75ef2c60f271ba
+121 a88d4228e2b765a7
+122 c031d1f22aee6a01
+123 d45baa1083ace06a
+124 cc17a43b3a4a6e4f
+125 817ac3d249b6e31c
+126 b881fa62bb18ee48
+127 5e07efbc7d04bb55
+128 b18a2ec47767e3ea
+129 d1ed35fdebea6e58
+130 a047266698d31a53
+131 4fb516b747cbccc0
+132 2d6d17dec0954a6d
+133 66808f2418324a1a
+134 48efedb5b253134b
+135 b0f98de5dd104363
+136 c4698dec98a3b339
+137 c8ae289ce21533fc
+138 d6997aa47c594c80
+139 5a178e34d9b21a57
+140 4c5e3c8f97ec3cc8
+141 7cfb8ffee781c1ba
+142 dc386e8eebdb7b86
+143 874c90c7521ed015
+144 fd115b4b9bb78acc
+145 11e28ff9c52b87c1
+146 03f2a8fbdf75c985
+147 595e3306b6aac20f
+148 326f6da251665bb9
+149 92705ca405543602
+150 d37bb564e6fa3ef0
+151 37068f078c239dac
+152 fce826e33fb60b04
+153 4a413bebc74d90f8
+154 8bb44e25d520734e
+155 7a8367e0f43b6a41
+156 958f1a0a500afa22
+157 a1d546727976c08c
+158 3f1746c8ded29613
+159 a47d9e2d7dbc69a2
+160 d85ee5e00e911e3c
+161 6b58ba0e5fe4630a
+162 f06c5b1495aefcb9
+163 6031d1fb224da944
+164 d1944fcdddfbb4be
+165 a1998899fa884878
+166 34e8191897464f45
+167 baf145cbf443d8ef
+168 fdef420a0141103a
+169 4fb73a6c63e78738
+170 8fdeb99397c79268
+171 784c091d317eec98
+172 ae77b76bdb1b336f
+173 ad0fd1f266f16d78
+174 c5fb27cca80fb484
+175 0199355c80c38a3d
+176 7abc63c1ba828956
+177 0a3476c069939e93
+178 b682c36402813bb1
+179 8ef6728cb32bc203
+180 cf5d88fb1ad7eae7
+181 55f65d346eb476bf
+182 49b5e58d4c1043bc
+183 91d295c6f9791d8d
+184 69cbae27f4704b8c
+185 a5d074681b6c33f9
+186 550ee440e7b12c14
+187 6a712c7f457b4ea5
+188 3b1d52c75aa4f91d
+189 beab83c97c68d7d8
+190 7d2db21db2df3e22
+191 e2d946ab301608b6
+192 943948021586a410
+193 0ab105605a5e6b3a
+194 b6c8ccd45d3daf2c
+195 5a8f158466639391
+196 f4e3cf734fa7155d
+197 c0af28c3dc9c72d1
+198 2e50fcd5b5773288
+199 84d39d3c95e3877b
+200 df6095d2fcf63abf
+201 b1f381d44ea560c6
+202 1d46071bdfa5ce53
+203 2096c66fbb727edb
+204 e06e305676f1eff3
+205 239dfa93982fac03
+206 9f45cdd129287f80
+207 76146142b07728e8
+208 1ad03ab51bd0b498
+209 cc271e30cb2ee743
+210 dc316c05ef4e574c
+211 b0cd4a8111acb078
+212 71a0ecf8379646cb
+213 a3794f2bf47077b3
+214 27df42fb986032d0
+215 aeabca8b1f3f9560
+216 3df559083a9c43f9
+217 b5868d5d7ce07eaf
+218 69ae83b51a5af98a
+219 ef3867687aff4fe2
+220 6fd2041d0aaa73c1
+221 4b9299c013637dcd
+222 9c2df4cc5d1de275
+223 d06a991174d65981
+224 ffd10c1f467715d4
+225 876a9a27a4a2962d
+226 e2249d55397d15c5
+227 cfdaec765c5ce0f5
+228 64654b31baf38ff7
+229 250026ebe66ec736
+230 febc9d8603a5afd6
+231 3db307eda565f531
+232 d043b02a156d3e6b
+233 d0cf7c5b3bf90b4a
+234 9d93e42783006917
+235 354abcd53a572480
+236 9b1bc521f8224028
+237 1fd6b0a80b5b0e02
+238 eea74b06fa42e337
+239 46bfd7e9c0f4a256
+240 761ec0ed218eafab
+241 be73b70f0d0d0eb6
+242 384d1851a8d081cc
+243 742339f2601f2e97
+244 86cc1593d9f9ea8f
+245 66678ab9f8bd814e
+246 b5da6f0b647466c3
+247 c53a75370fff953f
+248 12e2e9e392438acb
+249 de1d1481db6d9ae9
+250 e103e201c031c7b2
+251 cb29b22b97d7bb86
+252 b6c0ff3d5fda048e
+253 d94aa3060ca15d50
+254 bcd0db480242229b
+255 7ed4baf4dc9cb297
+256 1c4243cb88f3c3db
+257 8c278a8249c53883
+258 525e4caec9cf09cd
+259 40217ec1092ba46d
+260 19641fe430821dc4
+261 c808c80ebea45d24
+262 aabab54617a701e8
+263 6bea05a3a505515f
+264 b0f2c6e465596ed6
+265 249cb731d2fc22fc
+266 62f62c2ba8ec61ed
+267 c2ae74814c16eacf
+268 a2f1672ae9ef2004
+269 56da2d3ec3e17e1b
+270 cc2eb3ffcc711549
+271 f487c954624eec27
+272 340bcfb3f57f263d
+273 64c79ba28b88917f
+274 c3b91bc8360adf66
+275 d10d3001472328fd
+276 edaed753a773073a
+277 a6687292e8fa5321
+278 104b63d1d48187f6
+279 693165042a70ffe3
+280 48604ede4b134ef1
+281 98dda9f788b34961
+282 84f3ac3f6445ec3c
+283 032fa496df914742
+284 3b4d007dc422d369
+285 7fd40c3d52a11335
+286 3b265d24acd13d2e
+287 4c3017f8f3484a39
+288 e478a80b6b509800
+289 322fe0c265939c0c
+290 483b93a2979a733f
+291 deafb12aa4df5416
+292 1a3aedd3006fec8a
+293 1b07a3f972d80b7d
+294 8f7b5658eb5a64d2
+295 0bd94c2a0e0806ea
+296 5df166aa876cc132
+297 d16f3ccd9eef7a55
+298 5c23baa0b25b0027
+299 78e5b827673e876d
+300 30e867f1651cfc42
diff --git a/goldens/router_ceiling.log.bin b/goldens/router_ceiling.log.bin
new file mode 100644
index 0000000..b4d2f77
Binary files /dev/null and b/goldens/router_ceiling.log.bin differ
diff --git a/goldens/router_ceiling.t1 b/goldens/router_ceiling.t1
new file mode 100644
index 0000000..90d4b5b
--- /dev/null
+++ b/goldens/router_ceiling.t1
@@ -0,0 +1,87 @@
+# t1 manifest — per-tick FNV-1a-64 state hashes (full sim state)
+t1 1
+demo router_ceiling
+seed 42
+logic_hz 20
+catalog_hash a7b85c806df0397c
+ticks 80
+1 62efc7098f7f8e11
+2 ea6d25a5c3b60bde
+3 e314b640b31f92c7
+4 23892b305b4ea064
+5 8ed7b54c490a204d
+6 2afa6909a07f907a
+7 5565c778e8de5593
+8 18b3f016dbcfadb0
+9 7c5ebc1e1bac4410
+10 cbde2c75e4a1f989
+11 b33d27e560d20577
+12 4478ee54528a0488
+13 332fd7cb84853f39
+14 85bc1193f3e8cc8a
+15 518adbf3d40998e3
+16 31e5545a0069f89c
+17 f615c26611c60c85
+18 00583b72e52f49d6
+19 077040e310c9257f
+20 f26db6691677bf28
+21 0e572ea068da57b1
+22 9a4242911e922302
+23 27a74b09d8b853db
+24 53d69aa0ec851e34
+25 6354f545848b073d
+26 d59f75d8a098b4ee
+27 cb8906bef526a5b7
+28 0629c86d1046c920
+29 8474d9320eff6b89
+30 b0f1c37502682f7a
+31 c563e92827886ad3
+32 56a9db991867578c
+33 f2b93876932a5a35
+34 5061cf8551b2b7c6
+35 ac3cfb7cabf3d0ef
+36 6e3ce79a61af61d8
+37 6325c3266b1bdfe1
+38 49d53be1b92c48f2
+39 d03d9cb782e7358b
+40 ea907515f3ffb124
+41 184a116d457ae96d
+42 8d56211bf7ba8d5e
+43 a2c17c316de07fe7
+44 13c574eca022de90
+45 4c7beb0233850779
+46 7b63e18ad70a482a
+47 87b1ad6d91367e83
+48 7b4715aab14409bc
+49 bea0c1e49a6c0fa5
+50 1d988cbb8f1fc8f6
+51 de6206fc2509f29f
+52 a09a8301c8377348
+53 2084f713646964d1
+54 0b10b4028f568a22
+55 8e356792ed81c77b
+56 498aabeb5f3734d4
+57 f8e6fb066626d1dd
+58 8561aeeea93c920e
+59 c25db594fc3f7357
+60 b44d741f165631c0
+61 bd00604be77311a9
+62 69e74b4d8704e39a
+63 59e5b31b87830173
+64 e79563603ac818ac
+65 c4d0ce74b46cd3d5
+66 07a43bfb5a0a9366
+67 05dc41dbc2f5000f
+68 df2055b10282ad78
+69 b107cfd14e125381
+70 79f3e7cc545b0692
+71 7875cd789d74fd2b
+72 cb3f53913c7348c4
+73 8d900c28bfa8d10d
+74 a0ceac774bca1afe
+75 a84f2eb76f789687
+76 a0401d88c75e5ab0
+77 c6c9f7f3e19df119
+78 49f5675491b21c4a
+79 73d0bf45dd72a123
+80 31ea56771b17fcdc
diff --git a/goldens/router_ceiling/02500ms.png b/goldens/router_ceiling/02500ms.png
new file mode 100644
index 0000000..a1be9fe
Binary files /dev/null and b/goldens/router_ceiling/02500ms.png differ
diff --git a/goldens/router_ceiling/03500ms.png b/goldens/router_ceiling/03500ms.png
new file mode 100644
index 0000000..a1be9fe
Binary files /dev/null and b/goldens/router_ceiling/03500ms.png differ
diff --git a/goldens/router_tiers.log.bin b/goldens/router_tiers.log.bin
new file mode 100644
index 0000000..41dc1cc
Binary files /dev/null and b/goldens/router_tiers.log.bin differ
diff --git a/goldens/router_tiers.t1 b/goldens/router_tiers.t1
new file mode 100644
index 0000000..77a3f06
--- /dev/null
+++ b/goldens/router_tiers.t1
@@ -0,0 +1,87 @@
+# t1 manifest — per-tick FNV-1a-64 state hashes (full sim state)
+t1 1
+demo router_tiers
+seed 42
+logic_hz 20
+catalog_hash a7b85c806df0397c
+ticks 80
+1 b45262e24688e10c
+2 d12d9bfc1318c189
+3 fdecb7937daf7862
+4 5898bda778ecebb7
+5 c190f2008a98b1c8
+6 718f64bfe8406f75
+7 335ce16b84fac49e
+8 de44c98667dab223
+9 7b1d517980252f43
+10 0ada5b1ff36cffe4
+11 3a09c29ee158ebb2
+12 0dd155e9f3405c5b
+13 ef4e619366c45438
+14 ac6bd7501b255e0f
+15 60b19ec473d61d73
+16 22775bac7ee1537a
+17 3248e39fb63a84a3
+18 1939088adde64b20
+19 dba6789a9e128075
+20 fd77323fde8a299a
+21 bea8bf058e306e37
+22 5f4b8dc440afc964
+23 7c95e2a3f4ceb269
+24 51a1a465aca274fe
+25 b148f9bb0919e9cb
+26 5bcb545b174b1cc8
+27 b66913510c6ae3bd
+28 bb3f3a47b2b14642
+29 461ad2b761e95d5f
+30 d563aad86d5b05cc
+31 ba2923f2da8503d1
+32 9ea5e4a52d1981a6
+33 1242a63fb6721bd3
+34 a35281b822a58ad0
+35 1de521979ea180e5
+36 2a4d05bca1c4bd8a
+37 4b05d122fa579da7
+38 2f92c096d74a9194
+39 e5872f721506f359
+40 947913103570e06e
+41 be1e3eaea9a761fb
+42 18117c7c886d1d78
+43 d7bef1c269d2a76d
+44 75034a071dfc36f2
+45 72c443a15dc3becf
+46 6cf30680a456fa7c
+47 56ecbe99758fa441
+48 e885af378e1fc296
+49 1c9312e31404d4c3
+50 37c32a3e59547740
+51 71d8a1a956602a95
+52 cce716634c0c52ba
+53 956559efeda65857
+54 305e4481dcd21d84
+55 ae50e92d330f4709
+56 3e576a63bfdca49e
+57 fb9b6bb8ab83ed6b
+58 c31000a6911e56e8
+59 a62759d21181e25d
+60 463e17d477ea97e2
+61 d2e8f7e1eeda487f
+62 a390140ea8c22eec
+63 ec6c4e57539bb371
+64 51f25a1453e73fc6
+65 b7b6c9a414e3a673
+66 b36f58a86db8c770
+67 fa991561fd73dd05
+68 d8890c7b5734ea2a
+69 23085fa28e4d3247
+70 b4804b8484bd0034
+71 080e31162a0afbf9
+72 2eb62f0210cb390e
+73 c0c3d60a2ba08a9b
+74 e9a26375d1e8dc18
+75 92e20155dee2d70d
+76 9522015b3cc56812
+77 bf812c2a4ca5396f
+78 0e924afecf56439c
+79 554f99621d83bfe1
+80 335bb27ea91d82b6
diff --git a/goldens/router_tiers/02500ms.png b/goldens/router_tiers/02500ms.png
new file mode 100644
index 0000000..1e758eb
Binary files /dev/null and b/goldens/router_tiers/02500ms.png differ
diff --git a/goldens/router_tiers/03500ms.png b/goldens/router_tiers/03500ms.png
new file mode 100644
index 0000000..1e758eb
Binary files /dev/null and b/goldens/router_tiers/03500ms.png differ
